code/crates/core-consensus/src/effect.rs

```diff
@@ -133,6 +133,8 @@ where
     Decide(
         CommitCertificate<Ctx>,
         VoteExtensions<Ctx>,
+        malachitebft_core_driver::EvidenceMap<Ctx>,
+        malachitebft_core_votekeeper::EvidenceMap<Ctx>,
```

Comment:
Let's define our own `MisbehaviorEvidence` type in `core-consensus` which both evidence maps as fields.